src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -451,15 +533,46 @@ def add_job(  # pylint: disable=too-many-arguments
         step.jobs[job_id] = Job(
             status=status,
             index=index,
-            data=data,
             start_time=start_time,
             end_time=end_time,
             name=name,
             stdout=stdout,
             stderr=stderr,
+            current_memory_usage=current_memory_usage,
+            max_memory_usage=max_memory_usage,
         )
         return self
 
     def add_metadata(self, key: str, value: Any) -> "SnapshotBuilder":
         self.metadata[key] = value
```

Comment:
add_metadata is not called in the tests. Is it no longer used?

Reply:
it's gone.